### PR TITLE
Add Community Group and Corresponding Contribute Overview

### PR DESCRIPTION
**Pull request by Faolain**
**Additions:**
- Community Group for Contribution to live under which will in the future include more information regarding hackathons, yAcademy and others.
- Contributor's Page see: https://github.com/iearn-finance/docs/issues/85

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# Contribute
+
+## Yearn Community
```
Comment (milkyklim) [inline community/README.md:3]: There is no `community` rather `contributors`.

File: community/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+### File a Bug
+
+If a yearn product isn't working like you think it should please visit the appropriate Github repository listed in the [General Resources](##general-resources) section below or search in the [Yearn Github Repo](https://github.com/iearn-finance/yearn-protocol) if the issue already exists. If it does not exist please create a new issue following the issue template present in the repository(repo). If no issue template is present please include enough information such as your browser version, device, etc for that project so that the developers can replicate it and fix it. If the issue already exists you can signal your support by adding a thumbs up to the already existing issue or add any additional helpful information.
+
+### Leave Feedback
+
+Currently we use [Canny](https://yearnfinance.canny.io/) to raise general feedback.
```
Comment (milkyklim) [inline community/README.md:23]: Canny is dead.

File: community/README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+You can additionally see our ongoing progress on our [Github Project Board](https://github.com/orgs/iearn-finance/projects/1) and you can immediately begin contributing where you see fit for anything that's tagged as `help wanted` as long as you stick to the respective repository's `Contribute` page with its coding guidelines (linting, formatting, PRs, etc) It's that simple 🙂. You can also join us over at the #dev channel on [Discord](https://discord.com/invite/6PNv2nF) if that's more your thing.
+
+If you are interested in working on a particular project check out the list of Yearn products in the [General Resources](##general-resources) section below or if you want to directly check out our list of repos head on over to our [Docs: Code Repositories](https://docs.yearn.finance/developers/code-repositories).
+
+If you want a general overview of how our various services interact with one another for Vaults please see the schema and accompanying descriptions at **[Vaults Overview](https://docs.yearn.finance/developers/yvaults-documentation/vaults-overview)**. For an interactive schema strategy take a look at [yDAI]() to see how all the pieces of the smart contracts work together.
```
Comment (milkyklim) [inline community/README.md:53]: `[yDAI]()` empty link.

File: community/README.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+Lastly, if something is missing here that you think would make this better please create an issue here or submit a PR :)
+
+#### Designers
+
+If you are a UI/UX designer, animator, or graphic designer Yearn needs your help!
+
+Whether it is creating graphics to describe how our the Yearn systems such as strategies work, making Yearn easier to navigate, or simply adjusting misaligned text on our homepage, we welcome all designers aboard the Yearn ship.
+
+You can find us over on [Discord](https://discord.com/invite/6PNv2nF) at the #ui-ux channel and you can see our work log on our [Kanban Board](TODO: Create Design Project Board).
```
Comment (milkyklim) [inline community/README.md:67]: `TODO:` not done!

File: community/README.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+- [Yearn Snapshot](https://yearn.snapshot.page/)
+- [Yearn Newsletter](https://yearn.substack.com/) - Weekly YFI newsletter run by [@yfi_nomad](https://twitter.com/yfi_nomad)
+
+#### Social
+
+- Twitter:
+  - [Official Twitter Account ](https://twitter.com/iearnfinance)
+  - [Andre Cronje](https://twitter.com/AndreCronjeTech)
+  - [Klim (CEO of YFI)](https://twitter.com/milkyklim)
+  - [Blue Kirby](https://twitter.com/bluekirbyfi)
```
Comment (milkyklim) [inline community/README.md:120]: 🍴

**Review by milkyklim — CHANGES_REQUESTED** [review]
Need more polishing.

File: community/README.md (hunk 1 above)
Comment (Faolain) [reply community/README.md:3]: hmm so this was generated via gitbook under Community (which we had talked about on the call would be the place for contribution docs to live similar to Maker's Community Portal). Does this structure not make sense with this mind for gitbook?

File: community/README.md (hunk 2 above)
Comment (Faolain) [reply community/README.md:23]: Hmm I can remove but whenever I wrote the initial iteration it seemed to have been alive. I see there are still posts on there but can remove from this doc and replace with https://gov.yearn.finance/c/feedback/2

File: community/README.md (hunk 5 above)
Comment (Faolain) [reply community/README.md:120]: this section shows its age haha been sitting around for a while, will remove (was the source of the original links)

File: community/README.md (hunk 4 above)
Comment (Faolain) [reply community/README.md:67]: Removing for now as I don't see demand for UI/UX board 🤷‍♂️

File: community/README.md (hunk 3 above)
Comment (Faolain) [reply community/README.md]: Removed, only static image exists which can be confusing, will re-add with animation if/when finished or applicable.

**Comment by Faolain** [comment]
Updated

**Review by milkyklim — APPROVED** [review]

**Pull request merged**